addons/mail/static/tests/qunit_suite_tests/models/thread_tests.js

```diff
@@ -26,18 +26,18 @@ QUnit.test('inbox & starred mailboxes', async function (assert) {
 });
 
 QUnit.test('create (channel)', async function (assert) {
-    assert.expect(23);
+    assert.expect(20);
```

Comment:
why remove asserts in this PR?

Reply:
Some `assert`s were unnecessarily redundant once the test was modified to directly use `Channel` instead of `Thread`.